test/test-utils.js

```diff
@@ -0,0 +1,17 @@
+import sinon from 'sinon'
+
+export const isPropTypeWarningMatch = (warning) => {
+    return typeof(warning) === 'string' &&
+        warning.indexOf('Warning: Failed prop type:') > -1
+}
+
+
+export const stubConsoleError = () => {
```

Comment:
Don't abstract before/after blocks from their test context, this is important info which should be within your specs.